MAESTRO/html/scATAC_template.html

```diff
@@ -329,4 +335,4 @@ <h3 class="h5">
     </script>
 </body>
 
-</html>ƒ
\ No newline at end of file
+</html>
```

Comment:
Missing newline at the end of file

Reply:
I will add a newline at the end of the file. Thanks!